OpenRA.Mods.Common/Traits/Render/WithTileFootprintOverlay.cs

```diff
@@ -15,17 +15,17 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	public class BibInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>
+	public class WithTileFootprintOverlayInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>
```

Comment:
We could make this much more general purpose by defining a Size and Offset here, and then we can remove the Minibib stuff (which is just a normal bib but with a different size and offset).